### PR TITLE
Update community supported README: legal disclaimer

### PR DESCRIPTION
**Pull request by cwarnermm**
Added legal-approved disclaimer (bolded).

Preview
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # :warning: Announcements 
 
-Effective September 15th, 2023, the Focalboard plugin will no longer be bundled with Mattermost and will transition to being fully community supported. 
+Effective September 15th, 2023, the Focalboard plugin is no longer bundled with Mattermost and has transitioned to being fully community supported. 
+
+**This repository is community supported and not maintained by Mattermost. Mattermost disclaims any and all liability for integrations, including Third Party Integrations and Mattermost Integrations. All integrations are provided "AS IS", and may be used at your own risk. Integrations may be modified or discontinued at any time.**
 
 Self-hosted Mattermost instances may continue to use the Focalboard plugin without interruption; however, Mattermost developers will not be adding any new enhancements or bug fixes beyond September 15th, 2023. This Focalboard repository will remain open indefinitely for contributions from the open source community.
```
Comment (esethna) [inline README.md:7]: Effective September 15th, 2023, the Focalboard plugin is no longer bundled with Mattermost. Self-hosted Mattermost....

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
 # :warning: Announcements 
 
-Effective September 15th, 2023, the Focalboard plugin will no longer be bundled with Mattermost and will transition to being fully community supported. 
+Effective September 15th, 2023, the Focalboard plugin is no longer bundled with Mattermost and has transitioned to being fully community supported.
```
Comment (esethna) [inline README.md:3]: propose removing this and moving down to minimize duplication with the legal statement

**Review by esethna — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
 # :warning: Announcements
```
Comment (esethna) [inline README.md:1]: Should this be "Disclaimer" rather than "Announcements"?

**Review by esethna — APPROVED** [review]

**Review by sbishel — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,14 +1,12 @@
-# :warning: Announcements 
+# :warning: Disclaimer
```
Comment (cwarnermm) [inline README.md:1]:
```suggestion
# Disclaimer
```

File: README.md [hunk 5]
```diff
@@ -1,14 +1,12 @@
...
 
-Effective September 15th, 2023, the Focalboard plugin will no longer be bundled with Mattermost and will transition to being fully community supported. 
+**This repository is community supported and not maintained by Mattermost. Mattermost disclaims any and all liability for integrations, including Third Party Integrations and Mattermost Integrations. All integrations are provided "AS IS", and may be used at your own risk. Integrations may be modified or discontinued at any time.**
```
Comment (cwarnermm) [inline README.md:3]:
```suggestion
**This repository is community supported and not maintained by Mattermost. Mattermost disclaims liability for integrations, including Third Party Integrations and Mattermost Integrations. Integrations may be modified or discontinued at any time.**
```

**Comment by cwarnermm** [comment]
Legal's updated guidance and language for the legal disclaimer for community-supported integrations has been applied to this PR.

**Review by esethna — APPROVED** [review]
Thanks Carrie!

**Pull request merged**